package.json

```diff
@@ -50,10 +50,12 @@
     "create-project": "cross-env ts-node --swc scripts/create-project",
     "depcheck": "lerna exec --no-bail --stream -- depcheck",
     "dev": "npm run dev-docker && concurrently -n agones,server,client,taskserver -c '#CEB793,#FF9800,#7E2D40,#B2B4B2' npm:dev-agones-silent npm:dev-server npm:dev-client npm:dev-taskserver",
+    "dev:debug": "npm run dev-docker && concurrently -n agones,server,client,taskserver -c '#CEB793,#FF9800,#7E2D40,#B2B4B2' npm:dev-agones-silent npm:dev-server npm:dev-client-debug npm:dev-taskserver", 
     "dev-noclient": "npm run dev-docker && concurrently -n agones,server,taskserver npm:dev-agones-silent npm:dev-server npm:dev-taskserver",
     "dev-agones": "cd scripts && ./start-agones.sh",
     "dev-agones-silent": "npm run dev-agones &> /dev/null",
     "dev-client": "cd packages/client && npm run dev",
+    "dev-client-debug": "cd packages/client && cross-env VITE_DEBUG_MODE=true npm run dev",
```

Comment:
@HexaField do you think its a good idea to have it in a script or maybe use `.env.local` to toggle this behavior?

Reply:
yes